tests/integration/cases/urllib3_invalid_expectation/policy.dl

```diff
@@ -7,15 +7,15 @@ Policy("test_policy", component_id, "") :-
     check_passed(component_id, "mcn_build_as_code_1"),
     check_passed(component_id, "mcn_build_script_1"),
     check_passed(component_id, "mcn_build_service_1"),
+    check_passed(component_id, "mcn_version_control_system_1"),
     check_passed(component_id, "mcn_provenance_available_1"),
+    check_passed(component_id, "mcn_provenance_derived_commit_1"),
+    check_passed(component_id, "mcn_provenance_derived_repo_1"),
     check_passed(component_id, "mcn_provenance_level_three_1"),
-    check_passed(component_id, "mcn_version_control_system_1"),
     check_failed(component_id, "mcn_infer_artifact_pipeline_1"),
-    check_failed(component_id, "mcn_provenance_derived_commit_1"),
-    check_failed(component_id, "mcn_provenance_derived_repo_1"),
     check_failed(component_id, "mcn_provenance_witness_level_one_1"),
```

Comment:
I think the `mcn_provenance_expectation_1` should fail, but somehow it is not included in the policy. Can you please add that?